custom_components/opnsense/sensor.py

```
@@ -41,322 +42,370 @@
-                native_unit_of_measurement = None
-                icon = None
-                enabled_default = False
+    return entities
 
-                # state class
-                if "_kilobytes_per_second" in prop_name:
-                    state_class = SensorStateClass.MEASUREMENT
 
-                if prop_name == "connected_client_count":
-                    state_class = SensorStateClass.MEASUREMENT
+async def _compile_opnenvpn_server_sensors(
```

Ruff

`_compile_opnenvpn_server_sensors` is too complex (13 > 10)

(C901)